.gitlab-ci.yml

```diff
@@ -27,6 +28,10 @@ before_script:
   - if [[ -f "environment.$CI_COMMIT_REF_NAME" ]]; then
   -   source environment.$CI_COMMIT_REF_NAME
   - fi
+  - export ADMIN_USER_EMAILS=test@environment.com
```

Comment:
is this env var intentional? If so whats it doing?

Reply:
It is.  So, even though it sources `environment` above, it was getting a `KeyError` on `ADMIN_USER_EMAILS`.  I didn't want to set a default in the environment file itself, since it's only for testing, so I added it here.

Reply:
https://ucsc-ci.com/databiosphere/data-store/-/jobs/37778